Jenkinsfile

```
@@ -1,40 +1,9 @@
 #!/usr/bin/groovy
-/* Copyright (C) 2019-2022 Intel Corporation
- * All rights reserved.
+/*
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
```

I can't find an authoritative example for Groovy. However, according to the SPDX pages, it seems we are not doing it right:

> An SPDX short form identifier is a simple way to state the license that applies to a source code or documentation file.
>
> It consists of the following parts **on one line**:
>
> 1. [if required] The ‘begin comment’ characters for the applicable programming language (/*, //, #, …)
> 2. The characters SPDX-License-Identifier: with whitespace following the colon
> 3. An SPDX license ID (e.g., Apache-2.0) or SPDX license expression (see below)
> 4. [if required] The ‘end comment’ characters for the applicable programming language (*/, …)

So, I would say it has to be a comment in a single line. We can argue whether it should be a single-line comment or a multi-line comment e.g.

```suggestion
// SPDX-License-Identifier: BSD-2-Clause-Patent
```
or
```suggestion
/* SPDX-License-Identifier: BSD-2-Clause-Patent */
```

Ref: https://spdx.dev/learn/handling-license-info/#how (Format)
Ref: https://groovy-lang.org/syntax.html

---

We agreed offline with @grom72 to apply as follows:

```groovy
// SPDX-License-Identifier: BSD-2-Clause-Patent
// Copyright (c) 2019-2024 Intel Corporation.
```

---

+1 
An empty line between `// Copyright (C) 2019-2024 Intel Corporation.` and next comment, code:
```
// SPDX-License-Identifier: BSD-2-Clause-Patent
// Copyright (c) 2019-2024 Intel Corporation

// Next comment in the file
```